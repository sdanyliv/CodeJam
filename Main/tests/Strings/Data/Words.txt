fred
pic2
pic100a
pic120
pic121
jane
tom
pic02a
pic3
pic4
1-20
pic100
pic02000
10-20
1-02
1-2
x2-y7
x8-y8
x2-y08
x2-g8
pic01
pic02
pic 6
pic   7
pic 5
pic5
pic 5 
pic 5 something
pic 4 else