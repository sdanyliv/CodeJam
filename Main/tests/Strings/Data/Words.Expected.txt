1-2
1-02
1-20
10-20
fred
jane
pic01
pic2
pic02
pic02a
pic3
pic4
pic 4 else
pic5
pic 5
pic 5 
pic 5 something
pic 6
pic   7
pic100
pic100a
pic120
pic121
pic02000
tom
x2-g8
x2-y7
x2-y08
x8-y8